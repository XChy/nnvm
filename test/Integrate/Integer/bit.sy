int main() {
    int a = 3, b = 2;
    putint(a & b);  //2
    putint(3 & 2);
    putint(a | b);  //3
    putint(3 | 2);
    putint(a ^ b);  //1
    putint(3 ^ 2);
    putint(b << 1); //4
    putint(2 << 1);
    putint(b >> 1); //1
    putint(2 >> 1);
    putint(~b);
    putint(~2);
    return 0;       //0
}