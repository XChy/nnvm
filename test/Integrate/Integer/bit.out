2233114411
0